membership-common/src/main/scala/com/gu/memsub/subsv2/reads/ChargeListReads.scala

```diff
@@ -179,7 +180,17 @@ object ChargeListReads {
     }
   }
 
-  implicit def readSingle[B <: Benefit : ChargeReads]: ChargeListReads[ChargeList with SingleBenefit[B]] =
+  implicit def readSupporterPlusV2ChargeList: ChargeListReads[SupporterPlusCharges] = new ChargeListReads[SupporterPlusCharges] {
+
+    override def read(cat: PlanChargeMap, charges: List[ZuoraCharge]): ValidationNel[String, SupporterPlusCharges] = {
+      val pricingSummaries = charges.map(_.pricing)
+      Validation.success[NonEmptyList[String], SupporterPlusCharges](
+        SupporterPlusCharges(Month, pricingSummaries),
```

Comment:
@rupertbates this code path is running for pretty-much _all_ subs, not just S+, and so everything's being reported as monthly bill cycle in MMA.